packages/nextjs/app/page.tsx

```diff
@@ -14,7 +22,7 @@ const Home: NextPage = () => {
           <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
           <p className="text-lg flex gap-2 justify-center">
             <span className="font-bold">Checked in builders count:</span>
-            <span>To Be Implemented</span>
+            <span>{Number(checkedInCounter) || "Invalid"}</span>
```

Comment:
A couple of things here:

- It shows "Invalid" while loading the data from the contract. You can get `isLoading` from the read contract hook and tweak the logic (maybe show a [spinner](https://daisyui.com/components/loading/) or something?)
- Doing Number() in a `bigint` can cause overflow issues. Not in this case because the number will never be big, but it can happen when dealing with wei, etc. In this case, using .toString() might be safer.